libs/net/services/Actions/IngestAction.cs

```diff
@@ -93,7 +93,15 @@ public IngestAction(IApiService api, IOptions<TOptions> options, ILogger<IngestA
         if (reference != null)
         {
             if (reference.Status != (int)status) reference.Status = (int)status;
-            reference = await this.Api.UpdateContentReferenceAsync(reference, Headers);
+            try
```

Comment:
You're not logging anything extra here.  We already get an error that highlights the update failed.

Reply:
Updated, thanks